Formula/trzsz-go.rb

```diff
@@ -25,10 +25,10 @@ def install
 
   test do
     assert_match "trzsz go #{version}", shell_output("#{bin}/trzsz --version")
-    assert_match "trz (trzsz) go #{version}", shell_output("#{bin}/trz --version")
-    assert_match "tsz (trzsz) go #{version}", shell_output("#{bin}/tsz --version")
+    assert_match "trz (trzsz) go #{version}", shell_output("#{bin}/trz --version 2>&1")
+    assert_match "tsz (trzsz) go #{version}", shell_output("#{bin}/tsz --version 2>&1")
 
-    assert_match "executable file not found", shell_output("#{bin}/trzsz cmd_not_exists 2>&1", 255)
+    assert_match "spawn pty failed: &exec.Error", shell_output("#{bin}/trzsz cmd_not_exists 2>&1", 255)
```

Comment:
@lonnywong might need better handling for this.

Reply:
@chenrui333 Thanks. I'll try another way to test it later.

Reply:
trzsz-go: update test https://github.com/Homebrew/homebrew-core/pull/136820